WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -11,12 +18,12 @@ @implementation AccountServiceRemoteREST
 
 - (id)initWithApi:(WordPressComApi *)api
```

Comment:
I'm sorry, i know this wasn't introduced in this PR. But maaaaybe... can we change to *instancetype* instead?